l10n_fr_intrastat_service/data/intrastat_service_reminder.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2013-2017 Akretion (http://www.akretion.com/)
+  @author Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+<data noupdate="1"> <!-- noupdate = 1 for the 'active' field -->
+<record id="intrastat_service_cron" model="ir.cron">
```

Comment:
I don't think so, because the "noupdate" field is a boolean of ir_model_data, so it can only have 1 value per xmlid.